### ISSUE TITLE
Perception freezes after playerdm gain+drop

### ISSUE THREAD

**Issue by pF-arQon** [Beamdog/nwn-issues#635]
**To Reproduce**

Play the game, and promote yourself to DM. From that point on, encounter creatures will (randomly?) never be visible again, distant tiles will never render, even when later brought into range. Log out of DM mode and continue playing, and all those things will still be true, along with the minimap also never updating.

**Specifics**

 - Linux 6.5, 8193.36.12, Steam
 - A whole PW's worth of custom content, including the tilesets I was on and some creature, though basegame creatures in a different area exhibit the same bug.

**If needed, describe the bug**

Basically, once you gain and drop DM privs, there's a very good chance that nothing will ever update: new spawns are invisible, and even tiles (and minimap) that were out of PVS at the time are permanently black. going DM and back again will generally cause new spawns to start rendering reliably and local monsters to reappear, but the area geom is dead until you leave the area and come back.

Original discord thread with lots of pictures here: https://discord.com/channels/382306806866771978/1020053227472953445/threads/1105773847292162059
